Multiple swipers on the same view

vongohren opened the pull request:
Its not possible now to have multiple swipers in the same container
I cant re render the component and expecting the content to change.
We have a issue where we have a slider that is dependant on an dropdown value.
So when this value changes, the slider content changes.

But since the component is already mounted, there is not an entrance to newing up an new Swiper.

This is my suggetstion, works for us.

vongohren committed:
Multiple swipers on the same view

Its not possible now to have multiple swipers on the same view.
I cant re render the component and expecting the content to change.
We have a issue where we have a slider that is dependant on an dropdown value.
So when this value changes, the slider content changes.

But since the component is already mounted, there is not an entrance to newing up an new Swiper.

This is my suggetstion, works for us.

voronianski commented on react-swipe.js:
```diff
@@ -47,7 +47,8 @@
       callback        : React.PropTypes.func,
-      transitionEnd   : React.PropTypes.func
+      transitionEnd   : React.PropTypes.func,
+      newSwiper       : React.PropTypes.bool
```
@Snorlock I think should be a better name for this. Maybe `reinitSwipeOnUpdate`?

vongohren replied:
@voronianski  done

voronianski commented on react-swipe.js:
```diff
@@ -55,6 +56,9 @@
     },
 
     componentDidUpdate: function () {
+      if(this.props.newSwiper) {
```
@Snorlock  `if(` - space is missing.

vongohren replied:
done

vongohren committed:
Feedbacks on PR and added kill

Could not use setup because continious did not get updated accordingly to the props. So if you got a swiper that had one picture, the continuous flag would be set false and stay false since setup would not use any new props.

Kill had to be used because there were mulitple listeners which created problems for swiping.

voronianski commented:
@Snorlock please rebase to be able to merge.

voronianski commented:
@Snorlock I'm not able to merge this because `This branch has conflicts that must be resolved` could you rebase with latest master?

vongohren commented:
yeah il do it now. Its just that I used the online editor.

vongohren commented:
Done

vongohren committed:
alignin

voronianski commented:
@Snorlock :+1:
